packages/harpguru-core/src/components/menu-open-button/menu-open-button.tsx

```diff
@@ -0,0 +1,55 @@
+import Animated, { add } from 'react-native-reanimated'
+import { TapGestureHandler } from 'react-native-gesture-handler'
+import { View, StyleSheet } from 'react-native'
+import React from 'react'
+
+import { getMenuStylesAndAnimationVals } from '../../utils'
+import type { MenuProps } from '../../types'
+
+import { useTapAnimation } from './hooks'
+
+type LocalMenuProps = MenuProps & {
```

Comment:
This is one of those occassions when a `ChildrenProps` would be really useful. Perhaps it could be defined in line then instead of having to come up with this `Local...` abomination.

Reply:
Done